### ISSUE TITLE
Accessibility Concerns - Low Visibility of Focus Ring Indicators on Navbar & Buttons

### ISSUE THREAD

**Issue by ChalkandFeather** [hjtrhodes/bald-bible#63]
**Issue**
The focus ring indicators on navbar & button elements are currently difficult to perceive, potentially impacting the accessibility and usability of the application. When elements receive focus, users, especially those with visual impairments or relying on keyboard navigation, may struggle to identify the focused element due to the barely visible focus ring indicators.

**Steps to Reproduce:**
Navigate through various elements using keyboard navigation.
Observe the visibility of the focus ring indicator when elements receive focus.
Assess whether the focus ring is easily discernible, especially against the background or surrounding elements.

**Expected Behaviour**
Clearly Visible Contrast:
The focus ring should have sufficient contrast with the background or surrounding elements, making it easily distinguishable.
Distinctive Styling: Use a styling that is distinctive from the regular element appearance, such as a border or outline, to clearly indicate focus.
Chosen styling adheres to accessibility standards, considering factors beyond colour alone. This is crucial for users who may not perceive colour well or at all.
Consistency in the styling of focus rings across different types of elements. This helps users establish a predictable pattern when navigating through the interface.
Visible on Keyboard Navigation focus ring should be visible not only when navigating with a mouse but also when using keyboard navigation. This is particularly important for users who rely on keyboard input.

**Actual Behaviour**
Blue Focus ring barely visible on button elements. focus around form fields is good

**Recommendations:**
Increase the contrast or intensity of the focus ring to enhance visibility. Use blindsafe colours and test for WCAG 2.2 compliance
Consider using a different colour or style for the focus ring to make it stand out more prominently.
Ensure that the chosen design adheres to accessibility standards and does not rely solely on colour for differentiation.
Test the updated focus ring initially with plug-ins, then users who have diverse visual abilities to ensure improved visibility for a wide range of users.

**Impact**
Reduced accessibility and usability for users who rely on keyboard navigation or have visual impairments.
Potential frustration for users attempting to interact with the application.

Sign up button displays narrow margin blue border
<img width="1280" alt="Screenshot 2024-01-10 at 22 04 09" src="https://github.com/hjtrhodes/bald-bible/assets/114813829/08fe40f8-6f9e-49c6-9e7b-d02495fe5427">
With hover on sign up button, the style changes revealing broken narrow blue border
<img width="1280" alt="Screenshot 2024-01-10 at 22 04 21" src="https://github.com/hjtrhodes/bald-bible/assets/114813829/22c0af38-047c-47b9-b8af-f722b8d9bd3b">

Sign up button to submit form displays narrow margin blue border
<img width="1280" alt="Screenshot 2024-01-10 at 22 04 42" src="https://github.com/hjtrhodes/bald-bible/assets/114813829/871cd2df-8cbf-42a7-873b-85d532fd1573">
With hover on sign up button, the style changes revealing broken narrow blue border
<img width="1280" alt="Screenshot 2024-01-10 at 22 04 45" src="https://github.com/hjtrhodes/bald-bible/assets/114813829/723082fa-b49a-4f06-b417-075671bd2bc6">

**Comment by ChalkandFeather** [comment]
Issue also present with Log-in button

Log In button displays narrow margin blue border
<img width="1280" alt="Screenshot 2024-01-10 at 22 10 17" src="https://github.com/hjtrhodes/bald-bible/assets/114813829/95cc95e6-36c3-4707-bf51-0b67472fb99c">

With hover on Log in button, the style changes revealing broken narrow blue border 
<img width="1280" alt="Screenshot 2024-01-10 at 22 10 25" src="https://github.com/hjtrhodes/bald-bible/assets/114813829/3f15182b-5dc6-4f1f-b917-5c4d46af9049">

Log In button displays narrow margin blue border within form field
<img width="1280" alt="Screenshot 2024-01-10 at 22 11 29" src="https://github.com/hjtrhodes/bald-bible/assets/114813829/4012b1db-12b9-4a47-bfc8-a07ee57bf1d1">

With hover on Log in button, the style changes revealing broken narrow blue border within form field
<img width="1280" alt="Screenshot 2024-01-10 at 22 11 32" src="https://github.com/hjtrhodes/bald-bible/assets/114813829/b376b645-29db-4943-9531-90fb4ccf24d5">
